### DISCUSSION TITLE
header card items alignment

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1285] [Ideas]
WDYT about allowing left alignment for `header_card()`'s items or adding a tertiary items parameter? This would allow making headers that have the look and feel of H2O AI Cloud's header.

![image](https://user-images.githubusercontent.com/64787868/157084835-d4e16b69-7721-47ef-9d67-88b924063d95.png)

**Comment by mturoci** [comment]
Probably will not be added as this is purely cosmetic request and we currently focus on more impactful functional improvements.

**Reply by azim-b** [reply]
I was just wondering about the usefulness of this and was curious whether there's a reason why items can be aligned centred and right but not left (as of today).

I understand that this may not be an impactful improvement as much as some of the functional ones, no problems.
